training/tests/integration/config/test_config.yaml

```diff
@@ -1,12 +1,9 @@
 # Modifications for the basic template "config.yaml"
-hardware:
-  files:
-    dataset: anemoi-integration-tests/training/datasets/aifs-ea-an-oper-0001-mars-o48-1979-19-6h-v6-testset.zarr
-  paths:
-    data: https://object-store.os-api.cci1.ecmwf.int/ml-tests/test-data/samples
```

Comment:
Where would this path be set in the future?

Reply:
This path is not really needed in core in any case, as the functionality to get the test data lives in anemoi-utils and there you can see this path is defined https://github.com/ecmwf/anemoi-utils/blob/da380be71d78274d72bd0a3859ef00b1c80e9469/src/anemoi/utils/testing.py#L25